docs/DeveloperGuide.adoc

```diff
@@ -136,23 +136,23 @@ Each of the four components
 For example, the `Logic` component (see the class diagram given below) defines it's API in the `Logic.java` interface and exposes its functionality using the `LogicManager.java` class.
 
 .Class Diagram of the Logic Component
-image::LogicClassDiagram.png[width="800"]
+image::[width="800"]
```

Comment:
Do you mean to remove this image? 
```suggestion

```

Reply:
Should be good now

Reply:
And from what I can see, the current Developer Guide even without this merge already has this diagram updated from another PR. So should be fine either way

Reply:
As in, this change will remove the image. Is this what you intend to do?

Reply:
Okay, NOW it should be good